docker-compose.yml

```diff
@@ -185,6 +183,7 @@ services:
             - ${PUBLIC_PORT_BROWSER}:80
             - ${PUBLIC_PORT_PATTERN_LIBRARY}:8082
             - ${PUBLIC_PORT_DUMMY_API}:8083
+            - ${PUBLIC_PORT_JATS_INGESTER}:8085
```

Comment:
Happy for this to be public now but we should consider long term whether this should be the case.

Reply:
Yes, in general there should be some authentication on it?